contents/css/_components.nav-list.css

```diff
@@ -10,11 +10,26 @@
   justify-content: center;
 }
 
-.c-nav-list--footer-global .c-nav-list__anchor {
+.c-nav-list__anchor {
   color: currentColor;
-  margin-left: var(--l-unit);
-  margin-right: var(--l-unit);
+  display: inline-block;
+  padding-top: var(--l-unit);
+  padding-right: var(--l-unit);
+  padding-bottom: var(--l-unit);
+  padding-left: var(--l-unit);
+  @media(--m) {
+    padding-left: calc(var(--l-unit) * 3);
+    padding-right: calc(var(--l-unit) * 3);
+  }
+  &:hover,
+  &:focus {
+    color: var(--c-blue-violet);
+  }
+}
 
+.c-nav-list__anchor--social {
```

Comment:
A minor thing, "footer" would be actually a better modifier for this, however I’m fine with the current wording.

Reply:
Ha, yeah good point – it's a bit confusing because right now we have the "nav" in the position of the footer, and the social links are below it…
Let's clean up the semantics of the page elements when we work on #19, and the navigation moves to the header!